File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.Grids.NUnit._VB/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.vb
﻿'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Test>
    Public Async Function EnsureSampleDataServiceReturnsGridDataAsync() As Task
        Dim actual = Await SampleDataService.GetGridDataAsync()

        Assert.AreNotEqual(0, actual.Count)
    End Function
    '}]}
End Class

File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.Grids.NUnit/Param_ProjectName.Core.Tests.NUnit/Tests_postaction.cs
﻿//{[{
using System.Threading.Tasks;
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.NUnit
{
    public class Tests
    {
        //^^
        //{[{

        // Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Test]
        public async Task EnsureSampleDataServiceReturnsGridDataAsync()
        {
            var actual = await SampleDataService.GetGridDataAsync();

            Assert.AreNotEqual(0, actual.Count());
        }
        //}]}
    }
}
